.bandit.yml

```diff
@@ -2,3 +2,6 @@ skips:
 - B101
 - B320
 - B410
+- B101
```

Comment:
This is a duplicate. And it may be best to keep things alphabetical.

Reply:
https://github.com/scrapy/parsel/pull/287/commits/047282b57396b5308d45517ec6bdd9531f184b45